### PR TITLE
forgot to create group with Federated login

### PR DESCRIPTION
**Pull request by sarifern**
Condominio name is displayed at login

### CODE REVIEW

File: src/app/components/add-edit-condominios/add-edit-condominios.component.ts [hunk 1]
```diff
@@ -2,111 +2,72 @@ import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
...
-
-      console.log("editar");
-     /*  this.apiRequest = {
+      this.service.addCondominio(this.condo).subscribe((res) => {
+        this.apiResponse = res;
+        this.update.emit({ resUpdate: true });
+      });
+    } else if (this.mission == 'Editar Condominio') {
+      console.log('editar');
+      /*  this.apiRequest = {
```
Comment (ovaldezb) [inline src/app/components/add-edit-condominios/add-edit-condominios.component.ts]: Remove unsed code

File: src/app/components/add-edit-condominios/add-edit-condominios.component.ts [hunk 2]
```diff
@@ -2,111 +2,72 @@ import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
...
-
     console.log(this.mission);
-    
-
-    if (this.mission == "Agregar Condominio") {
 
+    if (this.mission == 'Agregar Condominio') {
       console.log(this.condo);
-   /* this.apiRequest = {"Condominio": this.condosGroupForm.value.condominio,
+      /* this.apiRequest = {"Condominio": this.condosGroupForm.value.condominio,
```
Comment (ovaldezb) [inline src/app/components/add-edit-condominios/add-edit-condominios.component.ts]: Remove unused code

File: src/app/components/add-edit-condominios/add-edit-condominios.component.ts [hunk 3]
```diff
@@ -2,111 +2,72 @@ import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
...
+      console.log('entra Editar');
 
-    if (this.mission == "Editar Condominio"){
-
-      console.log("entra Editar")
-
-      
-
-    /*  this.condosGroupForm = this.condosInfo.group({
+      /*  this.condosGroupForm = this.condosInfo.group({
```
Comment (ovaldezb) [inline src/app/components/add-edit-condominios/add-edit-condominios.component.ts]: Remove unused code

**Review by ovaldezb — CHANGES_REQUESTED** [review]

File: src/app/components/add-edit-condominios/add-edit-condominios.component.ts (hunk 1 above)
Comment (sarifern) [reply src/app/components/add-edit-condominios/add-edit-condominios.component.ts]: taken out code

File: src/app/components/add-edit-condominios/add-edit-condominios.component.ts (hunk 2 above)
Comment (sarifern) [reply src/app/components/add-edit-condominios/add-edit-condominios.component.ts]: taken out code

File: src/app/components/add-edit-condominios/add-edit-condominios.component.ts (hunk 3 above)
Comment (sarifern) [reply src/app/components/add-edit-condominios/add-edit-condominios.component.ts]: taken out code

**Review by ovaldezb — APPROVED** [review]
lgtm

**Pull request merged**